packages/core/cli/src/scripts/createProject/index.ts

```diff
@@ -16,6 +17,7 @@ async function createProject(integration: string, targetPath: string): Promise<v
   const absoluteTargetPath = path.join(__dirname, targetPath);
   const nuxtConfigPath = path.join(absoluteTargetPath, 'nuxt.config.js');
   await removeMagicCommentsFromFile(nuxtConfigPath);
```

Comment:
**suggestion** shouldn't both (with `uncommentprojectOnly` be a part of a same fn? or at least same file

Reply:
For file, ok but I am not sure about the function

Reply:
I meant that dealing with magic comments should in createProject can be a single step 9represented as fn) but this step could contain 2 functions. However you do this my point is that these 2 features should be in the same place for maintainability reasons. If one will change the second one will most likely have to change as well

Reply:
I decided to merge these 2 files into `processMagicComments.ts` which contains `processMagicComments` function which just runs these 2